lib/tasks/plugins.rake

```diff
@@ -1,5 +1,18 @@
-desc "List Installed plugins"
-task :plugins => :environment do
-  puts 'Collecting plugin information'
-  Foreman::Plugin.all.map{ |p| puts p.to_s }
+namespace :plugins do
+  desc "List Installed plugins"
+  task :list => :environment do
+    puts 'Collecting plugin information'
+    Foreman::Plugin.all.map{ |p| puts p.to_s }
+  end
```

Comment:
> Could we change the root to 'plugin' then? There are a number of other plugin rake tasks that start with `plugin:` and it would be nice to be in the same namespace.

Sure, done.

-- 
Later,
 Lukas #lzap Zapletal